CONTRIBUTING.rst

```diff
@@ -6,23 +6,23 @@ Contributing
 #. Make sure that your commit messages clearly describe the changes.
 #. Send a pull request.
 
-Here are some guidelines for hacking on gcloud-python.
+Here are some guidelines for hacking on ``gcloud-python``.
```

Comment:
@tseaver I can undo this bit if you like, I just noticed that `gcloud-python` was plain text in a lot of places in this file so I did a find and replace.